lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -245,6 +247,8 @@ void DAP::SendJSON(const std::string &json_str) {
   output.write_full(llvm::utostr(json_str.size()));
   output.write_full("\r\n\r\n");
   output.write_full(json_str);
+
+  LLDB_LOGV(GetLog(DAPLog::Transport), "{0} <-- {1}", name, json_str);
```

Comment:
should the `Transport` messages also include the client name? Otherwise, we will have a hard time understanding the logs if we have multiple clients connected

Reply:
that could be said for pretty much every log coming from lldb-dap. 
It wouldn'be a bad idea to have some new macros like DAP_LOGV that also receive a DAP instance and append the client name in the message

Reply:
Another thought that I have is that in an ideal world it would be nice to a logging category per client, so that you could do something like `lldb log enable lldb-dap transport my-client-1`, which would only enable logging for this specific client. You could potentially write to a different log file per client. 
Anyway, maybe this is not needed by anyone yet, but just leaving this idea here for posterity.